crypto/cmp/cmp_server.c

```diff
@@ -376,6 +416,61 @@ static OSSL_CMP_MSG *process_certConf(OSSL_CMP_SRV_CTX *srv_ctx,
     return msg;
 }
 
+/* pollreq should be handled sepreately, to avoid recursive call */
+static OSSL_CMP_MSG *process_request(OSSL_CMP_SRV_CTX *srv_ctx,
+                                     const OSSL_CMP_MSG *req)
+{
+    OSSL_CMP_MSG *rsp = NULL;
+
+    if (srv_ctx == NULL || srv_ctx->ctx == NULL
+            || req == NULL || req->body == NULL) {
+        ERR_raise(ERR_LIB_CMP, CMP_R_NULL_ARGUMENT);
+        return NULL;
+    }
```

Comment:
Checking these again is not needed, given the context in which this local function is called.
Better convert this to an `ossl_assert()` call.